services/launchpad/launchpad.tester.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('../service-tester');
+
+const t = new ServiceTester({ id: 'launchpad', title: 'Launchpad.net' })
+module.exports = t;
+
+t.create('gimp ppa version')
+  .get('/ppa/version/otto-kesselgulasch/gimp/gimp.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'version',
+    value: Joi.string()
```

Comment:
I suggest to use one of our existing version validators (https://github.com/badges/shields/blob/master/services/test-validators.js) or creating a new one. `Joi.string()` is too general in my opinion.